pkg/build/controller/build/build_controller.go

```diff
@@ -241,24 +261,26 @@ func (bc *BuildController) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer bc.buildQueue.ShutDown()
 	defer bc.buildConfigQueue.ShutDown()
-
-	// Read additionalCA data, if it exists
-	if len(bc.additionalTrustedCAPath) > 0 {
-		caData, err := bc.readBuildCAData()
-		if err != nil {
-			glog.Warningf("Failed to read additional CA bundle %s: %v", bc.additionalTrustedCAPath, err)
-		}
-		bc.additionalTrustedCAData = caData
-	}
+	defer bc.controllerConfigQueue.ShutDown()
```

Comment:
Instead of mounting the additional trusted CA bundle, we populate the additional CAs map from a separate control loop.